src/Runner.Listener/MessageListener.cs

```diff
@@ -397,5 +510,69 @@ private bool IsSessionCreationExceptionRetriable(Exception ex)
                 return true;
             }
         }
+
+        private async Task<VssCredentials> GetNewOAuthAuthorizationSetting(CancellationToken token)
+        {
+            Trace.Info("Start checking oauth authorization url update.");
+            while (true)
+            {
+                var backoff = BackoffTimerHelper.GetRandomBackoff(TimeSpan.FromMinutes(30), TimeSpan.FromMinutes(45));
+                await HostContext.Delay(backoff, token);
+
+                try
+                {
+                    var v2AuthorizationUrl = await _runnerServer.GetRunnerAuthUrlAsync(_settings.PoolId, _settings.AgentId);
+                    if (!string.IsNullOrEmpty(v2AuthorizationUrl))
+                    {
+                        var credData = _configStore.GetCredentials();
+                        var clientId = credData.Data.GetValueOrDefault("clientId", null);
+                        var currentAuthorizationUrl = credData.Data.GetValueOrDefault("authorizationUrl", null);
+                        Trace.Info($"Current authorization url: {currentAuthorizationUrl}, new authorization url: {v2AuthorizationUrl}");
+
+                        if (string.Equals(currentAuthorizationUrl, v2AuthorizationUrl, StringComparison.OrdinalIgnoreCase))
```

Comment:
this means when user configure the runner, service already make the runner using Token service, so there is no needed for migration.

Reply:
the authorization url is original return from service during runner config